rmgpy/molecule/pathfinder.py

```diff
@@ -480,3 +480,51 @@ def is_atom_able_to_lose_lone_pair(atom):
     return (((atom.is_nitrogen() or atom.is_sulfur()) and atom.lone_pairs in [1, 2, 3])
             or (atom.is_oxygen() and atom.lone_pairs in [2, 3])
             or atom.is_carbon() and atom.lone_pairs == 1)
+
```

Comment:
Do you know whether we actively "eradicate" radical electrons in an adsorbed molecule, e.g., by turning them into a lone pair, and same for partial charges? I guess not, since this will add a net charge to the molecule and RMG does not support that yet, right? Let's discuss this offline